testing/end_to_end_test.go

```diff
@@ -654,6 +654,20 @@ func (s *EndToEndTestSuite) createBatch(buildID string, experienceIDs []string,
 			Value: metricsBuildID,
 		})
 	}
+	if len(metricsBuildID) > 0 {
```

Comment:
```suggestion
	if len(metricsBuildID) > 0 {
```

This repeated block looks like a copy-paste error